.travis.yml

```diff
@@ -58,20 +75,28 @@ script:
   - mkdir ci-build && cd ci-build
   - cmake $BUILD_SWITCHES ..;
   - make
-  - if [ "$PUBLISH_BINARY" == "true" ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then
+  - if [ "$PUBLISH_BINARY" == "true" ] && [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$TRAVIS_MINGW" != "true" ]; then
       sudo make install;
     fi
-  - make rust-unit-tests
-  - ./ja2 -unittests
-  - ./ja2-launcher -help
-  - if [ "$PUBLISH_BINARY" == "true" ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then
+  - if [ "$TRAVIS_MINGW" != "true" ]; then
+      make rust-unit-tests;
+    fi
+  - if [ "$TRAVIS_MINGW" != "true" ]; then
+      ./ja2 -unittests;
+    fi
+  - if [ "$TRAVIS_MINGW" != "true" ]; then
+      ./ja2-launcher -help;
+    fi
```

Comment:
These three could be in the same block, but it's not that important. The diff looks fine, so merge away, unless you want to wait for testers/testing.

Reply:
Merged them...

Reply:
Are you sure they don't need to be and-chained (eg. make failing, launcher passing -> build passing)?

Reply:
True. Fixed it on master...